pkg/controller/kogitoapp/kogitoapp_controller.go

```diff
@@ -67,10 +150,9 @@ func (r *ReconcileKogitoApp) Reconcile(request reconcile.Request) (reconcile.Res
 	}
 
 	// create resources in the cluster that not exists
-	kogitoInv, err := inventory.CreateResources(&inventory.BuilderContext{
```

Comment:
This is old code, but on 145, you can simplify by using if "instance.Spec.Runtime != v1alpha1.SpringbootRuntimeType" and nothing else. You can keep the QuarkusRuntimeType check if you think that will be too confusing, but certainly don't need to check against empty.

Reply:
Absolutely